### PR TITLE
postgres to snowflake with cdc

### PR DESCRIPTION
**Pull request by rmoff**
Local environment to use for the Postgres/Snowflake blog post. 

Note that passwords are either dummy (`Welcome123`) or a private key that was created only for sample usage and has now been destroyed.

### CODE REVIEW

File: postgres-to-snowflake-with-cdc/postgres/replication-config.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ALTER TABLE omd_customers REPLICA IDENTITY FULL;
+ALTER TABLE omd_pets REPLICA IDENTITY FULL;
+ALTER TABLE omd_appointments REPLICA IDENTITY FULL;
+
+SELECT oid::regclass, relreplident FROM pg_class
+ WHERE oid in ( 'omd_customers'::regclass, 'omd_pets'::regclass, 'omd_appointments'::regclass);
```
Comment (gunnarmorling) [inline postgres-to-snowflake-with-cdc/postgres/replication-config.sql]: What's the purpose of having this select here?

File: postgres-to-snowflake-with-cdc/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    # and either put it directly in the file here, or write it to a .env file in 
+    # the same folder as this Docker Compose file in the form
+    # NGROK_AUTH_TOKEN=your_token_value
+    command: tcp postgres:5432 --log stdout --authtoken $NGROK_AUTH_TOKEN
+    ports:
+      - 4040:4040 # Web dashboard for ngrok
+
+  postgres:
+    container_name: postgres
+    image: quay.io/debezium/example-postgres:2.3
```
Comment (gunnarmorling) [inline postgres-to-snowflake-with-cdc/docker-compose.yml]:
```suggestion
    image: quay.io/debezium/example-postgres:3.0
```

**Review by gunnarmorling — APPROVED** [review]
LGTM overall. Two minor comments inline. Is there any reason for having folks do the scan, rather than just provide the connection resource definitions?

Very timely btw, there's a Reddit thread to which this is a perfect fit: https://old.reddit.com/r/dataengineering/comments/1gnipx1/tired_of_paying_fivetran_what_are_reliable/ (I've replied with some other resources already).

File: postgres-to-snowflake-with-cdc/postgres/replication-config.sql (hunk 1 above)
Comment (rmoff) [reply postgres-to-snowflake-with-cdc/postgres/replication-config.sql]: just to verify that it's worked. do you think I should remove it?

File: postgres-to-snowflake-with-cdc/docker-compose.yml (hunk 2 above)
Comment (rmoff) [reply postgres-to-snowflake-with-cdc/docker-compose.yml]: any compelling reason to change this? I'd rather ship an older version that I have tested.

**Comment by rmoff** [comment]
@gunnarmorling:

> Is there any reason for having folks do the scan, rather than just provide the connection resource definitions?

Two-fold: 

1. It's part of the workflow; not including it doesn't teach the user how to use the connectors when they want to run it for themselves 
2. `connection scan` generates a YAML which has the connection details embedded, so if you didn't get them to run `connection scan` you'd have to get them to edit the YAML, which is messy and goes even further again the above point

File: postgres-to-snowflake-with-cdc/postgres/replication-config.sql (hunk 1 above)
Comment (gunnarmorling) [reply postgres-to-snowflake-with-cdc/postgres/replication-config.sql]: Just a bit odd to have a select in a set-up script and not doing anything with the result. I'd either remove it, or put a comment clarifying its purpose. Or have sort of an assert which fails if it doesn't return the expected state (just kidding, don't do that ;)

File: postgres-to-snowflake-with-cdc/docker-compose.yml (hunk 2 above)
Comment (gunnarmorling) [reply postgres-to-snowflake-with-cdc/docker-compose.yml]: Yeah, it's an arbitrary old version which may make this example look outdated. If you want to stay on 2.x, I'd use the latest there (2.5 as of now), though I don't expect any issues with just moving to 3.0 (those example images don't change much between versions; arguably, they should not follow Debezium versions in the first place).

File: postgres-to-snowflake-with-cdc/postgres/replication-config.sql (hunk 1 above)
Comment (rmoff) [reply postgres-to-snowflake-with-cdc/postgres/replication-config.sql]: Done 👍

**Comment by gunnarmorling** [comment]
>It's part of the workflow

Ok, that's fair.

> you'd have to get them to edit the YAML,

I don't get that part yet, why is that? Couldn't you give the entire thing verbatim and folks could apply it? Just asking for my own education, it makes sense as-is for the other reason you state.

**Comment by rmoff** [comment]
> > you'd have to get them to edit the YAML,
> 
> I don't get that part yet, why is that? Couldn't you give the entire thing verbatim and folks could apply it? Just asking for my own education, it makes sense as-is for the other reason you state.

the connection details are in the YAML, and will be specific to each user's Postgres and Snowflake

**Comment by gunnarmorling** [comment]
Ah, ok, got it now, thanks. SF creds will be specific indeed.

**Pull request merged**